app/code/Magento/GoogleAnalytics/Block/Ga.php

```diff
@@ -131,7 +131,7 @@ public function getOrdersTrackingCode()
                         'quantity': %s
                     });",
                     $this->escapeJs($item->getSku()),
```

Comment:
Could you replace in all places `escapeJs` to `escapeJsQuote` where is't needed?

Reply:
Just done thanks